pinc/CharSuites.inc

```diff
@@ -8,12 +8,12 @@ class CharSuiteNotEnabledException extends Exception
 class PickerSet
 {
     public string $name;
-    /** @var array<string, (string|null)[]> */
+    /** @var (?string[])[] */
```

Comment:
The first string should not be null (as it was). But there should be another level of array since below
```php
$this->subsets[$name] = $codepoints;
```
Although it doesn't seem to make any difference

Reply:
I don't think I understand this comment. There are `null` entries in the inner array, see e.g. https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/charsuite-basic-greek.inc#L20

Did you mean something else I missed?

Reply:
Sorry, I was getting confused. Approving it now.